tools/run_docker.sh

```diff
@@ -33,3 +35,5 @@ do
 
     dock_ext_args=""  # Reset docker args, just in case
 done
+
+find dist -type f -not -name "*aiohttp*" | xargs rm -f
```

Comment:
I've covered this with https://github.com/aio-libs/aiohttp/pull/2334/files#diff-efcf4fcbe5802c1378b182a49aaf85e8R40 (tested with other repo, but should also work)

Reply:
Maybe you mean another link -- but let's keep the line as is for sake of quick merge.
You can polish the line later.
I'm striving to release aiohttp 2.3 ASAP

Reply:
find has `-delete` option, no need to pipe to xargs.

Reply:
yes, but the match expression is a bit more complete :)

Reply:
`find /io/dist -maxdepth 1 -type f ! -path '/io/dist/'"$package_name"'-*-manylinux1_${arch}.whl' -exec rm -fv {} \;` @asvetlov I'm referring to this line